File: JavaScript/BinaryTreeTilt/BinaryTreeTilt.js
/**
 * Definition for a binary tree node.
 * function TreeNode(val) {
 *     this.val = val;
 *     this.left = this.right = null;
 * }
 */
/**
 * (168ms)
 * @param {TreeNode} root
 * @return {number}
 */
var findTilt = function(root) {
    var tilt = 0;
    getSum = function(root) {
      if(root === null)  
        return 0;
      var left = getSum(root.left);
      var right = getSum(root.right);
      tilt += Math.abs(left - right);
      return root.val + left +right;
    };
    getSum(root);
    return tilt;
};

File: Python/BinaryTreeTilt/BinaryTreeTilt.py
# Definition for a binary tree node.
# class TreeNode(object):
#     def __init__(self, x):
#         self.val = x
#         self.left = None
#         self.right = None

class Solution(object):
	# (129ms)
    def findTilt(self, root):
        """
        :type root: TreeNode
        :rtype: int
        """
        def getSum(root, tilts):
            if not root:
                return 0
            left, right = getSum(root.left, tilts), getSum(root.right, tilts)
            tilts.append(abs(left - right))
            return root.val + left + right
        tilts = []
        getSum(root, tilts)
        return sum(tilts)